crates/bevy_internal/src/lib.rs

```diff
@@ -193,3 +190,40 @@ pub mod dynamic_plugin {
     //! Dynamic linking of plugins
     pub use bevy_dynamic_plugin::*;
 }
+
+/// `use bevy::prelude::*;` to import common components, bundles, and plugins.
+pub mod prelude {
+    #[doc(hidden)]
+    pub use crate::{
+        app::prelude::*, asset::prelude::*, core::prelude::*, core_pipeline::prelude::*,
+        ecs::prelude::*, hierarchy::prelude::*, input::prelude::*, log::prelude::*,
+        math::prelude::*, render::prelude::*, text::prelude::*, time::prelude::*,
+        transform::prelude::*, ui::prelude::*, utils::prelude::*, window::prelude::*,
+        DefaultPlugins, MinimalPlugins,
+    };
+
+    pub use bevy_derive::{bevy_main, Deref, DerefMut};
+}
+
+#[doc(hidden)]
+pub mod prelude_2d {
+    pub use crate::prelude::*;
+
+    #[cfg(feature = "bevy_core_pipeline")]
+    pub use crate::core_pipeline::prelude::{Camera2d, Camera2dBundle};
+    #[cfg(feature = "bevy_sprite")]
+    pub use crate::sprite::prelude::*;
+}
+
+#[doc(hidden)]
+pub mod prelude_3d {
+    pub use crate::prelude::*;
+
+    #[cfg(feature = "bevy_core_pipeline")]
+    pub use crate::core_pipeline::prelude::{Camera3d, Camera3dBundle};
+    #[cfg(feature = "bevy_pbr")]
+    pub use crate::pbr::prelude::*;
+
+    pub use crate::reflect::prelude::*;
```

Comment:
Is there a reason why this is only in 3d?

Reply:
Yes, it is (almost) only used in 3d